library/ui-styles/src/main/res/values/colors.xml

```diff
@@ -136,4 +136,9 @@
     <color name="shield_color_black">#17191C</color>
     <color name="shield_color_warning">#FF4B55</color>
 
+    <!-- Badge Colors -->
+    <attr name="vctr_badge_color_border" format="color" />
+    <color name="vctr_badge_color_border_light">#FFFFFF</color>
+    <color name="vctr_badge_color_border_dark">#21262C</color>
```

Comment:
Use `palette_black_950` instead of hard-coded color?

Reply:
In case it's not clear: we could have:
```xml
<color name="vctr_badge_color_border_dark">@color/palette_black_950</color>
```

Reply:
Correct